linux_os/guide/system/accounts/accounts-restrictions/account_expiration/policy_temp_passwords_immediate_change/rule.yml

```diff
@@ -53,3 +53,5 @@ ocil: |-
 
     <pre># chage -d 0 [UserName]
     # passwd -e [UserName]</pre>
+
+{{{ warning_rule_deprecated_cause("SLES 12 STIG Revision v2R10", release='0.1.69') | indent(4) }}}
```

Comment:
This deprecation cause is specific to SLES 12 STIG. I think it should be included in a Jinja2 conditional other wise the warning will be visible in other products which may use this rule. If so, this will confuse the users.